pkg/e2e/up_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
+// +build !windows
```

Comment:
This test does not compile on Windows due to:
```go
cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
```
and surrounding logic

Reply:
Ah yeah, we've dealt with this same thing before in Tilt 🙃 

What we can do is make a `func SetProcessPgid(cmd *exec.Cmd)` in `proc_windows.go` that's a no-op and `proc_unix.go` that does the code ⬆️ and then call the helper. Annoying but 🤷 

(IIRC Windows always has the desired behavior re: killing process groups)